app/game.js

```diff
@@ -174,10 +181,12 @@ class Game {
         participants[this.current].fold = true
         break
       case COMMANDS.CALL:
+        let user = participants[this.current];
         ok = this.bet(
-          participants[this.current],
+          user,
           this.currentBet - participants[this.current].bet
         )
+        user.allIn = user.balance <= 0;
```

Comment:
`this.bet` should handle the scenario that the user's balance isn't sufficient, so I think it can be `user.allIn = user.balance === 0`